app/controllers/root_controller.rb

```diff
@@ -1,5 +1,6 @@
 class RootController < ApplicationController
   skip_before_action :authenticate_user!
+  skip_before_action :verify_and_set_current_person
```

Comment:
You can remove this if you implement https://github.com/rubyforgood/homeward-tails/pull/1354/files#r2023859335